content/resources/user/gen3-client.md

```diff
@@ -137,55 +184,56 @@ Finished downloads/24d0be10-d164-48ad-aafa-9fcaac682df9 2570240 / 2570240 bytes
+Example of manifest.json (Minimal):
+```
+{
+  {
+    "object_id": "a12ff17c-2fc0-475a-9c21-50c19950b082"
+  },
+  {
+    "object_id": "b12ff17c-2fc0-475a-9c21-50c19950b082"
+  },
+  {
+    "object_id": "c12ff17c-2fc0-475a-9c21-50c19950b082"
+  }
+}
+```
 
-The manifest file should not contain headers and each row should be only a GUID and filename separated by a tab.
+The gen3-client will upload all the files in the provided manifest using the `gen3-client upload-manifest` command.
 
-Example:
-```
-a12ff17c-2fc0-475a-9c21-50c19950b082	filename-1.txt
-b22ff17c-2fc0-475a-9c21-50c19950b082	filename-2.txt
-c32ff17c-2fc0-475a-9c21-50c19950b082	filename-3.txt
+Example Usage:
 ```
-The following script text should be added to a file named 'gen3_manifest.sh', and the script should then be run in the directory containing the files. 
+gen3-client upload-manifest --profile=<profile_name> --manifest=<manifest_file> --upload-path=<path_for_files>
 
-``` bash
-#!/bin/bash
-cat $2 | while read line
-do
-f=`echo $line | tr -d '\r'`
-guid=$(echo $f | awk '{print $1}')
-fname=$(echo $f | awk '{print $2}')
-if [ -f "$fname" ]
-then
-	echo Uploading file $fname with GUID $guid
-	gen3-client upload --profile $1 --guid $guid --file $fname
-	echo Upload of $fname complete!
-else
-	echo $file not found.
-	exit 1
-fi
-done
+gen3-client upload-manifest --profile=bob --manifest=manifest.json --upload-path=upload
+
+Uploading data ...
+a12ff17c-2fc0-475a-9c21-50c19950b082  3.64 MiB / 3.64 MiB [==========================================================================================] 100.00%
+b12ff17c-2fc0-475a-9c21-50c19950b082  3.63 MiB / 3.63 MiB [==========================================================================================] 100.00%
+c12ff17c-2fc0-475a-9c21-50c19950b082  3.65 MiB / 3.65 MiB [==========================================================================================] 100.00%
+Successfully uploaded file "a12ff17c-2fc0-475a-9c21-50c19950b082" to GUID a12ff17c-2fc0-475a-9c21-50c19950b082.
+Successfully uploaded file "b12ff17c-2fc0-475a-9c21-50c19950b082" to GUID b22ff17c-2fc0-475a-9c21-50c19950b082.
+Successfully uploaded file "c12ff17c-2fc0-475a-9c21-50c19950b082" to GUID c22ff17c-2fc0-475a-9c21-50c19950b082.
+3 files uploaded.
 ```
 
 * * *
-## 7) Generate a metadata TSV
+## 8) Generate a metadata TSV
 * * *
 In order to register data files in a Gen3 data commons, the filenames, md5sums, and file_size in bytes, must be submitted as metadata. The gen3-client can help collect the values of these three properties using the `gen3-client generate-tsv` command.
 
 The template TSV for a data file node should be downloaded from the node's entry page in the data dictionary and used as a template with this command. Then the wildcard character `*` can be used to add all matching files to the specified output tsv.
```

Comment:
Maybe more detail on where to find the template to download here

Reply:
I put some more info there and an image. Now we have two versions of the dictionary page and they are quite different. In gen3.org there is nothing about the new dictionary view, but I think the new view has been deployed to some commons already, like NIAID. Should we take that into consideration here?